README.md

```diff
@@ -650,7 +650,61 @@ Chainweb.Pact.poll(request_keys, network_id: :testnet04, chain_id: 1)
    ]
  }}  
 ```
+### Listen endpoint
 
+Retrieves transaction results with a single key.
+
+```elixir
+Kadena.Chainweb.Pact.listen(request_key, network_opts \\ [network_id: :testnet04, chain_id: 0])
+```
+
+**Parameters**
+
+- `request_key`: A string. A request key is the unique id of a Pact transaction consisting of its hash, it is obtained from submitting a command via the  [Send endpoint](#send-endpoint).
```

Comment:
@EdwinGuayacan replace "A string." with "String value.", and remove space before "Send endpoint" ⬇️ 
```suggestion
- `request_key`: String value. A request key is the unique id of a Pact transaction consisting of its hash, it is obtained from submitting a command via the [Send endpoint](#send-endpoint).
```